praw/models/reddit/live.py

```diff
@@ -122,6 +122,71 @@ def __init__(self, thread):
         """
         self.thread = thread
 
+    def invite(self, redditor, permissions=None):
+        """Invite a redditor to be a contributor of the live thread.
+
+        Raise :class:`praw.exceptions.APIException` if the invitation
+        already exists.
+
+        :param redditor: A redditor name (e.g., ``'spez'``) or
+            :class:`~.Redditor` instance.
+        :param permissions: When provided (not ``None``), permissions should
+            be a list of strings specifying which subset of permissions to
+            grant. An empty list ``[]`` indicates no permissions, and when
+            not provided (``None``), indicates full permissions.
+
+        Usage:
+
+        .. code-block:: python
+
+           thread = reddit.live('ukaeu1ik4sw5')
+           redditor = reddit.redditor('spez')
+
+           # 'manage' and 'settings' permissions
+           thread.contributor.invite(redditor, ['manage', 'settings'])
+
+        :seealso: :meth:`.LiveContributorRelationship.remove_invite` to
+            remove the invite for redditor.
+
+        """
+        if permissions is None:
```

Comment:
Is there a common permission that could instead be used in place of `-access`? That would then possibly work for both. Or we can add a keyword argument that provides that value.